pennylane_lightning/core/src/lightning_qubit.toml

```diff
@@ -0,0 +1,125 @@
+schema = 1
+
+[device]
+name = "lightning.qubit"
+precision = ['float32', 'float64']
+
+[operations]
+observables = [
+	"NamedObs",
+	"HermitianObs",
+	"TensorObs",
+	"HamiltonianObs",
+]
+
+# These are gates which are defined in Quantum Instruction Set.
+[[operations.gates]]
+native = [
+        "QubitUnitary",
+        "PauliX",
+        "PauliY",
+        "PauliZ",
+        "MultiRZ",
+        "Hadamard",
+        "S",
+        "T",
+        "CNOT",
+        "SWAP",
+        "CSWAP",
+        "Toffoli",
+        "CY",
+        "CZ",
+        "PhaseShift",
+        "ControlledPhaseShift",
+        "RX",
+        "RY",
+        "RZ",
+        "Rot",
+        "CRX",
+        "CRY",
+        "CRZ",
+        "CRot",
+        "IsingXX",
+        "IsingYY",
+        "IsingZZ",
+        "IsingXY",
+        # Adjoint'ed gates
+        # which just need the inverse flag set
+        # are considered natively supported gates.
+        "Adjoint(S)",
+        "Adjoint(T)",
+]
+# decomp gates are those which are neither supported natively
+# and are not executed via the applyMatrix operation. Technically
+# they could also be decomposed via applyMatrix.
+decomp = [
+        "SX",
+        "ISWAP",
+        "PSWAP",
+        "SISWAP",
+        "SQISW",
+        "CPhase",
+        # Adjoint'ed gates
+        # which are not in native set must either be decomposed
+        # or applied via matrix.
+        "Adjoint(SX)",
+        "Adjoint(ISWAP)",
+        "Adjoint(SISWAP)",
+        "Identity",
+        "BasisState",
+        "QubitStateVector",
+        "StatePrep",
+        "ControlledQubitUnitary",
+        "DiagonalQubitUnitary",
+        "SingleExcitation",
+        "SingleExcitationPlus",
+        "SingleExcitationMinus",
+        "DoubleExcitation",
+        "DoubleExcitationPlus",
+        "DoubleExcitationMinus",
+        "QubitCarry",
+        "QubitSum",
+        "OrbitalRotation",
+        "QFT",
+        "ECR",
+]
+# QubitUnitary should never be here.
+# Gates in "matrix" are decomposed to QubitUnitary.
+# Having a QubitUnitary here would mean an infinite loop.
+
+# These are gates that appear on the list of supported gates
+# but are not in the list of Quantum Gate Set Instructions in
+# the Catalyst repository.
+# One disadvantage of matrix gates is that they will be lowered to
+# QubitUnitary and QubitUnitary is not differentiable.
+# TODO: Move some of these to native once we add support for them
+# in Catalyst runtime.
+matrix = [
+        "MultiControlledX",
```

Comment:
This gate exists in a PR to be merged soon: https://github.com/PennyLaneAI/pennylane-lightning/pull/516

Though, we may need to update the signature to what you will need to call it directly

Reply:
This is a misconception. This file relates more to Catalyst than lightning at the moment. When PR #516 is merged and when we create a corresponding `__qir__qis__MultiControlledX` we can move this gate to the list of "natively" supported gates. It is an interface issue that we are aware and are thinking to improve in the future.